src/cmds/edit.rs

```diff
@@ -22,6 +22,9 @@ use clap::{Command as ClapCommand, ArgMatches, Arg};
 /// ```
 pub struct EditCommand;
 
+pub const CODE_START: &str = r#"// @lc code=start"#;
+pub const CODE_END: &str = r#"// @lc code=end"#;
```

Comment:
shoud configure these in the config, the `//` can not be recognized by every programming languages

Reply:
Yeah, you are right, `//` maybe wrong!